docker/docker-entrypoint.sh

```diff
@@ -86,7 +86,7 @@ initialize() {
 		"${CONSUME_DIR}"; do
 		if [[ ! -d "${dir}" ]]; then
 			echo "Creating directory ${dir}"
-			mkdir "${dir}"
+			mkdir -p "${dir}"
```

Comment:
If we're not going to be typing it, I like to use the long option for quick readability for anyone.

Reply:
I used the short option because 6 lines below there is another `mkdir -p` with the short option. Should I change the other one below from `-p` to `--parents` along the way?

Reply:
That'd be great.  I haven't made an effort to find them all, just as noticed, making the changes